odoo-shippable/scripts/build-image.sh

```diff
@@ -422,7 +422,7 @@ PG_NON_DURABILITY=1 PG_LOGS_ENABLE=1 python ${REPO_REQUIREMENTS}/linit_hook/trav
 
 # Install & Configure RVM
 curl -sSL https://rvm.io/mpapis.asc | gpg --import -
-\curl -sSL https://get.rvm.io | /bin/bash -s stable --ruby
+\curl -sSL https://raw.githubusercontent.com/wayneeseguin/rvm/stable/binscripts/rvm-installer | /bin/bash -s stable --ruby
```

Comment:
In fact, docker hub is failing could you check what is wrong with the build?
https://hub.docker.com/r/vauxoo/odoo-80-image-shippable-auto/builds/